pytorch_lightning/core/datamodule.py

```diff
@@ -15,10 +15,9 @@
 
 import functools
 import inspect
-import os
 from abc import abstractmethod
 from argparse import ArgumentParser, Namespace
-from typing import Any, List, Optional, Tuple, Union, Dict, Sequence, Mapping
+from typing import Any, List, Mapping, Optional, Sequence, Tuple, Union
```

Comment:
isort was failing in CI because of import ordering in `datamodule.py`, not sure why since I did not originally modify the file at all. But since I had to fix the import order there, I also removed unused imports and applied yapf while I was at it.

Reply:
I also had to fix `test_datamodules.py`, I have no idea why :) There is an unused class `CustomMNISTDataModule` there by the way.

Reply:
> There is an unused class CustomMNISTDataModule there by the way.

Reply:
already removed.